resources/views/components/modal-trigger.blade.php

```diff
@@ -39,13 +39,13 @@ class="bg-box-bg relative w-full h-screen max-h-screen overflow-y-auto sm:h-auto
             >
                 <div class="flex items-center justify-between pb-2">
                     <p class="text-lg font-semibold mb-4">{{ $modalTitleLabel }}</p>
-                    <button 
```

Comment:
It looks like VS Code ignores the editorconfig for trimming whitespace?
https://github.com/RetroAchievements/RAWeb/blob/master/.editorconfig#L9
`trim_trailing_whitespace = true`

Reply:
Confirmed, VSCode does indeed ignore _.editorconfig_ unless a third-party extension is installed.